certified-connectors/Connect Bridge/apiDefinition.swagger.json

```diff
@@ -238,6 +250,18 @@
         "summary": "Execute SQL query",
         "description": "This action executes a SQL query after a connection being created",
         "parameters": [
+          {
+            "name": "format",
+            "in": "query",
+            "required": true,
```

Comment:
Please confirm that the existing flows for this connector will not break from this change.

The required parameter with default value has been added. ValuePath: operation/ExecuteQuery/parameter/format Source

Reply:
I confirm that the existing flows will not break from this change.

Actually this is a workaround because the policy stopped working for some reason.